src/esp/assets/FRLInstanceMeshData.h

```diff
@@ -55,6 +69,8 @@ class FRLInstanceMeshData : public GenericInstanceMeshData {
 
   // Gravity direction of the mesh, this is a STATIC
   vec3f orig_gravity_dir;
+
+  void compute_buffer();
```

Comment:
computeBuffer();

Also, would you please leave a comment on what you are doing in this function.

Reply:
Good catch! The function will be removed.